packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -1,13 +1,36 @@
+// @codepen
+
 import * as React from 'react';
 import { Link } from 'office-ui-fabric-react/lib/Link';
 import { DetailsList, IGroup, IGroupDividerProps } from 'office-ui-fabric-react/lib/DetailsList';
-import { createListItems, createGroups } from 'office-ui-fabric-react/lib/utilities/exampleData';
-import './DetailsListExample.scss';
+import { createListItems, createGroups, IExampleItem } from 'office-ui-fabric-react/lib/utilities/exampleData';
+import { mergeStyles, DefaultPalette, DefaultFontStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const headerClass = mergeStyles({
```

Comment:
When you have a bunch like this, it is more idiomatic (and convenient) to use `mergeStyleSets`

Reply:
Maybe this should be obvious, but how do I do something like this with mergeStyleSets?
```
const headerTitleClass = mergeStyles(DefaultFontStyles.xLarge, {
  padding: '4px 0'
});
```

Reply:
```ts
const classNames = mergeStyleSets({
  headerClass: { //... }, // argument for mergeStyles call in line 9
  headerTitleClass: {}, //argument for mergeStyles call in line 20 
 // etc. 
});
```

Reply:
Then you access via `classNames.headerClass`, `classNames.headerTitleClass`

Reply:
Thanks, but specifically what I meant was how do you merge in an existing style like `DefaultFontStyles.xLarge`